.gitignore

```diff
@@ -7,6 +7,9 @@ pids
 *.pid
 *.seed
 
+# Configuration for integration tests
+test/config.json
```

Comment:
This is not provided, which will disarm integration tests. Can be added locally to run against local (hermetic) server